torchrl/envs/transforms/transforms.py

```diff
@@ -1707,6 +1708,7 @@ def __init__(
         self,
         keys_in: Optional[Sequence[str]] = None,
         shared_td: Optional[TensorDictBase] = None,
+        lock: mp.Lock = (mp.Lock()),
```

Comment:
Can we do this instead

```python
...
    lock: Optional[mp.Lock] = None
...
if lock is None:
    lock = mp.Lock()
```

We usually try to avoid instantiating stuff in kwargs